assignments/this.js

```diff
@@ -13,14 +13,55 @@
 
 // code example for Window Binding
 
+function hello() {
+    console.log(this);
+}
+
+hello();
+
 // Principle 2
 
 // code example for Implicit Binding
 
+const wizard = {
+    name: 'Harry Potter',
+    spell: 'expelliarmus',
+    cast: function() {
+        console.log(`${this.name} casts ${this.spell}!`);
+    }
+};
+wizard.cast();
+
 // Principle 3
 
 // code example for New Binding
 
+function LevitationPartner(partner){
```

Comment:
one option to make your code more dynamic: function LevitationPartner(partner, incantation){}.  This would allow for different incantations to be passed in to the function rather than hard coding the value "Wingardium Leviosa"